server/src/main/java/org/eclipse/openvsx/UpstreamRegistryService.java

```diff
@@ -381,6 +381,26 @@ public String getPublicKey(String publicId) {
         }
     }
 
+    @Override
```

Comment:
It doesn't look like the method will be used by any use-case. Should it be some stub instead?

Reply:
I am also unaware of a use case for now.

We could also reconsider extending the interface if there is no use case. I extended the `IExtensionRegistry` interface because `getRegistryVersion()` fulfills the interface declares "[...] registry API methods that can be accessed without authentication".

So we have three possibilities:
- Remove from interface
- Stub with UnsupportedOperationException
- Keep implementation even if not used for now

Which one would you prefer?

Reply:
For now I would propose to keep it, but add a comment that the functionality is not used at the moment, but could be used if there is a need to show a version of an upstream registry.
In future I would propose to revise the interfaces, maybe remove/add more methods.

Reply:
Good idea, done!